examples/RenderManagerD3DCAPIExample.cpp

```diff
@@ -103,17 +104,12 @@ void RenderView(
     context->OMSetRenderTargets(1, &renderTargetView, depthStencilView);
 
     // Set up the viewport we're going to draw into.
-    CD3D11_VIEWPORT viewport(static_cast<float>(renderInfo.viewport.left),
-                             static_cast<float>(renderInfo.viewport.lower),
-                             static_cast<float>(renderInfo.viewport.width),
-                             static_cast<float>(renderInfo.viewport.height));
-    context->RSSetViewports(1, &viewport);
+	CD3D11_VIEWPORT viewport(static_cast<float>(eye == 0 ? 0 : renderInfo.viewport.width),
```

Comment:
This should be handled by RenderManager, not by the application.  The viewport should have been set correctly by the osvrRenderManagerGetRenderInfoFromCollectionD3D11() call.